src/sass/_storage.scss

```
@@ -83,3 +93,12 @@
     margin-top: -7px;
   }
 }
+
+.storage-pool-meter-container {
+  align-items: center;
+  display: flex;
+}
+
+.storage-pool-overview-meter {
+  display: flex;
+}
```

```suggestion
.cluster-member-with-meters {
  display: flex;
}
```
Better name and move this into the namespace for `storage-overview-tab`. This doesn't need to be on the root namespace.